src/screens/TopicPageScreen/elements/TopicDomainHeader.js

```diff
@@ -4,66 +4,63 @@ import {Pressable, StyleSheet, Text, View} from 'react-native';
 import SimpleToast from 'react-native-simple-toast';
 import CommunityIcon from '../../../assets/icon/CommunityIcon';
 import {Shimmer} from '../../../components/Shimmer/Shimmer';
-import {colors} from '../../../utils/colors';
 import {fonts, normalize, normalizeFontSize} from '../../../utils/fonts';
 import {convertString} from '../../../utils/string/StringUtils';
 import {COLORS} from '../../../utils/theme';
 
 const TopicDomainHeader = (props) => {
   const {domain, handleOnMemberPress, hideSeeMember, isFollow, memberCount} = props;
-  const isUserFollow = props?.initialData?.isFollowing ? props.initialData.isFollowing : isFollow;
 
   const handlePress = () => {
-    if (isUserFollow) {
+    if (isFollow) {
       handleOnMemberPress();
     } else {
       SimpleToast.show('Only community members can see other members', SimpleToast.SHORT);
     }
   };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `handlePress` function checks if the user is following the community before allowing them to see other members. This is a good security measure as it prevents non-members from accessing member information. However, it would be better to also handle this check on the server-side to ensure data privacy even if someone bypasses the client-side check.



<!-- This is an auto-generated comment by OSS CodeRabbit -->